argocd_observability/deployment_event.md

```diff
@@ -0,0 +1,37 @@
+### ArgoCD Sync Deployment event
+
+| Property | Source | in Dashboard used | 
+|---|---|---|
+| event.id  |   |
+| event.kind = "SDLC_EVENT" |   | x 
+| event.version = "0.1.0" | |  x
+| event.category = "task" | |  x
+| event.type = "deployment" | |  x
+| event.provider = "argocd" | |  x
+| event.status | |  x
```

Comment:
started/finished?

Reply:
Correct. 
The pipeline sets the value of `event.status` which is either `started` or `finished`.